modules/quant_purger/src/Entity/QuantPurgeSettings.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Drupal\quant_purger\Entity;
+
+use Drupal\purge\Plugin\Purge\Purger\PurgerSettingsBase;
+use Drupal\purge\Plugin\Purge\Purger\PurgerSettingsInterface;
+
+/**
+ * Defines the QuantPurgerSettings entity.
+ *
+ * @ConfigEntityType(
+ *   id = "quantpurgersettings",
+ *   label = @Translation("QuantCDN"),
+ *   config_prefix = "settings",
+ *   static_cache = TRUE,
+ *   entity_keys = {"id" = "id"},
+ *   config_export = {
+ *     "id",
+ *     "label",
+ *     "description",
+ *     "name",
+ *     "invalidationtype",
+ *     "runtime_measurement",
+ *     "timeout",
+ *     "connect_timeout",
+ *     "cooldown_time",
+ *     "max_requests",
+ *     "http_errors"
+ *   },
+ * )
+ */
+class QuantPurgeSettings extends PurgerSettingsBase implements PurgerSettingsInterface {
```

Comment:
Id is quantpurgersettings (with an "r") but the class is QuantPurgeSettings (without an "r") o_O

Reply:
The class should be updated to use the "r"